### PR TITLE
Feat: Add search placeholder option

### PR DESCRIPTION
**Pull request by Blakeinstein**
Adds an option to configure placeholder for the search bar.

Fixes: #780

### CODE REVIEW

File: src/types/options.ts [hunk 1]
```diff
@@ -30,6 +30,12 @@ export interface TreeConfiguration {
      */
     searchMode: SearchMode;
 
+    /**
+     * Placeholder shown for the search bar.
+     * (Default: `Search`)
```
Comment (ckotzbauer) [inline src/types/options.ts:35]: The default should be "" here as well.

**Review by ckotzbauer — COMMENTED** [review]
Thanks for that great PR. I only added one small comment.
Could you please rebase the PR? I hopefully fixed a bug in the Github-Actions in master. It's the first external PR to this project 😉

File: src/types/options.ts (hunk 1 above)
Comment (Blakeinstein) [reply src/types/options.ts:35]: Oh year right

**Comment by Blakeinstein** [comment]
Rebased and pushed

**Comment by ckotzbauer** [comment]
Ok, I really don't know why GH does not start the three jobs now 😞 
I will have a look later today and then cut the release.

**Review by ckotzbauer — APPROVED** [review]

**Comment by ckotzbauer** [comment]
Okay, I had to double-approve the runs. Pretty weired. 😆

**Comment by ckotzbauer** [comment]
Thanks for your contribution!

**Comment by Blakeinstein** [comment]
Happens with CI/CD! It requires to fail 3-4 times before it can work :p.

Any ETA on release?

**Comment by ckotzbauer** [comment]
yes, now 😉 
Released as 1.1.0 🎉

**Pull request merged**